landingzones/templates/landingzones/project_zones.html

```diff
@@ -161,6 +161,11 @@ <h3><i class="iconify" data-icon="mdi:briefcase-upload"></i> Landing Zones</h3>
     window.irodsShowChecksumCol = true;
   </script>
 
+  <!-- Superuser status variable -->
+  <script>
```

Comment:
Nice touch providing the URL for JQuery via the template, it's much better than hardcoding it.

We can include the URL in the previous `<script>` block which already provides some settings to the client side Javascript. No need to declare a new one.

Other than that, no notes for this pull request. Good work :)